Jenkinsfile

```diff
@@ -5,7 +5,7 @@ pipeline {
             steps {
                 sh 'rm -rf build && mkdir build'
                 dir('build') {
-                    sh 'cmake -DCMAKE_BUILD_TYPE=Debug -DUSE_SANITIZER=Address ..'
+                    sh 'cmake -DCMAKE_BUILD_TYPE=Debug -DUSE_SANITIZER=Address -DCMAKE_CXX_FLAGS="-DLOG_LEVEL=LOG_LEVEL_TRACE" ..'
```

Comment:
I'd not touch the Jenkinsfile for now, we have a bunch of changes to be incorporated shortly. They do include at least one trace build. After those are in.. if further changes are desired, you can suggest via PR

Reply:
Fine with me. I saw that in the other PR by crd, he has already done the same thing. I will remove this change.